docs/spelling_wordlist.txt

```diff
@@ -139,3 +139,4 @@ docstrings
 numpy
 jsonpickle
 crowdworkers
+AudioAnnotation
```

Comment:
You don't need to add this to spelling list if every time you refer to it you use backtip around it